src/status_im2/common/log.cljs

```diff
@@ -18,15 +20,22 @@
 
 (defn setup
   [level]
-  (when-not (string/blank? level)
-    (log/set-level! (-> level
-                        string/lower-case
-                        keyword))
-    (log/merge-config!
-     {:output-fn (fn [& data]
-                   (let [res (apply log/default-output-fn data)]
-                     (add-log-entry res)
-                     res))})))
+  (let [handle-error (fn [res]
+                       (let [{:keys [error]} (types/json->clj res)]
+                         (when-not (string/blank? error)
+                           (log/error "init statusgo logging failed" error))))]
+    (if (string/blank? level)
+      (native-module/init-statusgo-logging true false "WARN" handle-error)
```

Comment:
Small suggestion to use a map instead of positional args, this helps readers understand what `true` and `false` mean without having to jump to the source implementation.